test/Libraries/DynamoMSOfficeTests/ExcelTests.cs

```diff
@@ -490,38 +464,118 @@ public void CanWrite1DDataOfMixedTypesToExcelWorksheet()
             string openPath = Path.Combine(TestDirectory, @"core\excel\NewWorkbook_AddMixed1DData.dyn");
             ViewModel.OpenCommand.Execute(openPath);
             Assert.AreEqual(13, ViewModel.CurrentSpace.Nodes.Count);
+            
             var watch = ViewModel.Model.CurrentWorkspace.GetDSFunctionNodeFromWorkspace("Excel.GetDataFromExcelWorksheet");
             ViewModel.HomeSpace.Run();
 
             Assert.IsTrue(watch.CachedValue.IsCollection);
+            
+            var data = new object[] { new object[] { "doodle" }, new object[] { 0.000 }, new object[] { 21029.000 }, new object[] { null }, new object[] { -90.000 } };
+            AssertPreviewValue(watch.GUID.ToString(), data);
```

Comment:
Is there a way to validate, if the data is really written to xls file or not? Maybe after write is performed, read from the same file to validate the content. Also, you need to make sure that the newly written file is deleted at the end of the test.